micrometer-tracing-bridges/micrometer-tracing-bridge-brave/src/main/java/io/micrometer/tracing/brave/bridge/BraveBaggageInScope.java

```diff
@@ -77,10 +77,16 @@ public String get(TraceContext traceContext) {
     @Deprecated
     public Baggage set(String value) {
         if (this.traceContext != null) {
-            this.delegate.updateValue(this.traceContext, value);
+            boolean success = this.delegate.updateValue(this.traceContext, value);
+            if (logger.isTraceEnabled()) {
+                logger.trace("Managed to update the baggage on set [" + success + "]");
+            }
         }
         else {
-            this.delegate.updateValue(value);
+            boolean success = this.delegate.updateValue(value);
+            if (logger.isTraceEnabled()) {
+                logger.trace("Managed to update the baggage on set [" + success + "]");
```

Comment:
Would it be beneficial to distinguish the log from the situation where a `traceContext` is present?